src/sentry/seer/autofix/prompts/root_cause.py

```diff
@@ -0,0 +1,16 @@
+ROOT_CAUSE_PROMPT = """Analyze issue {short_id}: "{title}" (culprit: {culprit})
+
+Your task is to find the ROOT CAUSE of this issue. Do not propose fixes - only identify why the error is happening.
+
+Guidelines:
+1. Use your tools to fetch the issue details and examine the evidence
+2. Investigate the trace, replay, logs, other issues, trends, and other telemetry when available to gain a deeper understanding of the issue
```

Comment:
(for later) does transitioning to explorer mean we can encourage it to look at other events in the issue to capture a slightly broader picture? feel like it's nicer for RCA to not mention transient, event-specific info. maybe explorer will search events on its own if it needs to, since i see there's already prompting + tool descs around issue vs event